auslib/admin/views/forms.py

```diff
@@ -168,7 +168,7 @@ class PartialReleaseForm(Form):
 
 
 class RuleForm(Form):
-    backgroundRate = IntegerField('Background Rate', validators=[Required(), NumberRange(0, 100)])
+    backgroundRate = IntegerField('Background Rate', validators=[validators.any_of(range(0,100,1))])
```

Comment:
I made new pull request. I have tried to execute tests and confirmed that it doesn't work with old method, and that it works with new one.  I also noticed that I had some mistake in code so I fixed it. 

run-test.sh script had some failed stuff but I don't think they were related to my changes in code.